docs/Server.md

```diff
@@ -164,9 +164,20 @@ Please note this setting this option to `false` goes against
 ### `requestIdHeader`
 
 The header name used to know the request id. See [the request id](https://github.com/fastify/fastify/blob/master/docs/Logging.md#logging-request-id) section.
+When this option is specified along with [`genReqId`](https://github.com/fastify/fastify/blob/master/docs/Logging.md#factory-gen-request-id)
```

Comment:
This makes it sound like the "request-id" header only takes precedence over the `genReqId` function if this option is specified, which is a little unclear since this option is always "specified" because it has a default value. Perhaps this could be simplified to just say that if the header exists it will be used to set the request ID so `genReqId` won't be called.

Reply:
Can you provide the text you would like to see instead?

Reply:
Actually, instead of providing this information here, it would be simpler to just change the note under `genReqId` to say:

```
**Note: genReqId will _not_ be called if the 'request-id' header is available.**
```